assets/src/parse_bundle_plugin.js

```diff
@@ -44,13 +44,8 @@ var parseBundlePlugin = function(data, base_dir) {
             // instantiated as a global variable. Only currently used by the Kolibri core app.
             external = data.name;
             // change the periods of the Python module path name to underscores, so that it is a valid JS variable name.
-            library = data.core ? 'Kolibri' : data.name.replace(/\./g, "_");
+            library = data.core ? 'kolibriGlobal' : data.name.replace(/\./g, "_");
```

Comment:
I wanted to choose a more unique name to help with code search and debugging: the string `Kolibri` returns many results. Since we have variables being dynamically inserted into the client by build processes, I wanted to help make it clear where it came from.